spec/factories/claim/advocate_interim_claims.rb

```diff
@@ -1,12 +1,20 @@
 FactoryBot.define do
   factory :advocate_interim_claim, class: Claim::AdvocateInterimClaim do
-    court
-    case_number { random_case_number }
-    creator { build(:external_user, :advocate) }
-    external_user { creator }
+    advocate_base_setup
+    case_type nil
+
+    after(:build) do |claim|
+      claim.creator = claim.external_user
+    end
 
     trait :submitted do
-      state :submitted
+      after(:create) { |c| c.submit! }
+    end
+
+    trait :without_fees do
```

Comment:
**[not a blocker/remark]** This is one of the reasons I was suggesting not to use common Advocate factory setup as we're adding stuff to it by default that we then need to destroy. At least the advocate interim claim was "clean" enough that we could add things as required, not destroy stuff that was added by default that we don't actually need for a specific scenario.